stream/dub.sdl

```diff
@@ -14,19 +12,25 @@ importPaths "."
 copyFiles "../lib/win-i386/libeay32.dll" "../lib/win-i386/ssleay32.dll" platform="windows-x86"
 copyFiles "../lib/win-amd64/libeay32.dll" "../lib/win-amd64/ssleay32.dll" platform="windows-x86_64"
 
-configuration "windows-mscoff" {
+// LDC 1.0.0 doesn't include libz in libphobos
+libs "z" platform="posix-ldc"
+libs "z" platform="gdc"
+
+configuration "openssl-mscoff" {
```

Comment:
Hehe, I would this doesn't result in combinatoric explosion in the future, but I guess the "hack" would be to move SSL into separate sub-package if more configurations options are ever needed for `stream`. Maybe - on the long run - it makes sense to go an approach like [SecureD](https://github.com/LightBender/SecureD) and provide a common API over botan/openssl/libressl/... in a new "ssl" package?